docs/src/rsh/errors/index.md

```diff
@@ -2116,6 +2116,12 @@ Maybe(UInt).Some(5).match({
 This error indicates that you have inspected the details about a publication, such as via `{!rsh} didPublish()`, before there has been a publication.
 This is impossible, so the expression must be moved after the first publication.
 
+```reach
+  didPublish(handAlice)
+  Alice.publish(handAlice);
+  commit();
+```
+
```

Comment:
This entry is incomplete. 
We need to explicitly state that this sample is erroneous and then create a sample that is correct and explicitly state that the example is the proper way to use `didPublish()`